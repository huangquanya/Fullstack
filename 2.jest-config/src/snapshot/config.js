function config() {
  return {
    name: 'lilei',
    age: 456
  }
}


export {
  config
}