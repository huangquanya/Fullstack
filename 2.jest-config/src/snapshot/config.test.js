import { config } from './config';


describe('snapshot test', () => {

  test('config', () => {
    expect(config()).toMatchSnapshot();
  })
})