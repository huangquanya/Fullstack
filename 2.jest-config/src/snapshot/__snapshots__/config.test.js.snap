// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test config 1`] = `
Object {
  "age": 456,
  "name": "lilei",
}
`;
